package.json

```diff
@@ -12,7 +12,8 @@
     "lexus-spec": "./bin/lexus-spec"
   },
   "scripts": {
-    "lint": "eslint --ext .js ./"
+    "test": "npm run lint ; mocha --ui tdd --recursive test/",
+    "lint": "./node_modules/.bin/eslint ./ || exit 0"
```

Comment:
Can you re-order these just to use alphabetical?

Reply:
Also you'll need to add `exec('npm test')` after `exec('npm install')` in the `Jenkinsfile`.